### ISSUE TITLE
list Account ID in `ivpn account`

### ISSUE THREAD

**Issue by lsm5** [ivpn/desktop-app-cli#13]
# Feature request

## Description

It would be nice to see the Account ID in `ivpn account`.

## Describe the solution you'd like

```
$ ivpn account
Account ID: FOO
Plan:      FOO
Active until: FOO
```
## Describe alternatives you've considered

It's seen in the desktop-ui as well as the android apps, but not in the cli app AFAICT.

**Comment by lsm5** [comment]
Forgot to mention, I see this with:
```
$ rpm -q ivpn
ivpn-2.12.16-1.x86_64
```

Any ETA on the newer versions in the yum repo?

**Comment by stenya** [comment]
Thanks, @lsm5 . We are preparing a new release for Linux (it will also include the feature you requested). 
It will be released soon.

**Comment by stenya** [comment]
@lsm5  Requested functionality is released in v3.3.7.

**Comment by lsm5** [comment]
@stenya LGTM, thanks a lot!! :)
